m3/include/m3/thrift/gen.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+thrift --gen cpp --out . m3.thrift
```

Comment:
could you migrate this to use an executable docker image (similar to [this](https://github.com/m3db/m3db/blob/master/src/dbnode/generated/thrift/generate.sh)).

Reply:
Hmm, is it worth the overhead? The thrift output includes the version used and we can document it here to be more explicit.

Reply:
it's an annoyance for devs having to install the native thrift lib, but yeah the thrift code is going to be regen'd fairly rarely so w/e. i'm fine as is if you document the version clearly here